command-expansion-server/src/app.ts

```diff
@@ -433,6 +433,44 @@ export interface SeqBuilder {
   ): Sequence;
 }
 
+//generate a sequence JSON from a sequence file
+app.post('/get-sequence-edsl', async (req, res, next) => {
+  const commandDictionaryID = req.body.input.commandDictionaryID as number;
+  const edslBody = req.body.input.edslBody as string;
+
+  let commandTypes;
+  try {
+    const context: Context = res.locals['context'];
+    commandTypes = await context.commandTypescriptDataLoader.load({ dictionaryId: commandDictionaryID });
+  } catch (e) {
+    res.status(200).json({
+      message: 'Error loading command dictionary',
+      cause: (e as Error).message,
+    });
+    return next();
+  }
+
+  const result = await ((await piscina.run(
+    {
+      edslBody,
+      commandTypes,
+    },
+    { name: 'executeEDSL' },
+  )) as ReturnType<typeof executeEDSL>);
+
+  const sequenceJson = result.sequenceJson;
+  if (sequenceJson != null) {
+    res.status(200).json(sequenceJson);
+  } else {
+    res.status(200).json({
+      message: 'Failure to generate sequence JSON',
+      cause: result.errors,
+    });
+  }
+
+  return next();
+});
+
 app.post('/get-seqjson-for-sequence', async (req, res, next) => {
```

Comment:
Given the above comment about the new api endpoint, I think it might make sense to rename this one as well to be something more along the lines of `get-seqjson-for-seqid-and-simulation-dataset` or some less verbose version of that

Reply:
I can't think of a less verbose name so I changed it to `get-seqjson-for-seqid-and-simulation-dataset` which matches the new endpoint `get-seqjson-for-sequence-standalone`